plugins/Intl/lang/id.json

```diff
@@ -450,36 +500,72 @@
-        "LongMonth_10": "Oktober",
-        "LongMonth_11": "November",
-        "LongMonth_12": "Desember",
-        "LongMonth_2": "Februari",
-        "LongMonth_3": "Maret",
-        "LongMonth_4": "April",
-        "LongMonth_5": "Mei",
-        "LongMonth_6": "Juni",
-        "LongMonth_7": "Juli",
-        "LongMonth_8": "Agustus",
-        "LongMonth_9": "September",
         "Minutes": "menit",
+        "Month_Long_1": "Januari",
+        "Month_Long_10": "Oktober",
+        "Month_Long_11": "November",
+        "Month_Long_12": "Desember",
+        "Month_Long_2": "Februari",
+        "Month_Long_3": "Maret",
+        "Month_Long_4": "April",
+        "Month_Long_5": "Mei",
+        "Month_Long_6": "Juni",
+        "Month_Long_7": "Juli",
+        "Month_Long_8": "Agustus",
+        "Month_Long_9": "September",
+        "Month_Long_StandAlone_1": "Januari",
+        "Month_Long_StandAlone_10": "Oktober",
+        "Month_Long_StandAlone_11": "November",
+        "Month_Long_StandAlone_12": "Desember",
+        "Month_Long_StandAlone_2": "Februari",
+        "Month_Long_StandAlone_3": "Maret",
+        "Month_Long_StandAlone_4": "April",
+        "Month_Long_StandAlone_5": "Mei",
+        "Month_Long_StandAlone_6": "Juni",
+        "Month_Long_StandAlone_7": "Juli",
+        "Month_Long_StandAlone_8": "Agustus",
+        "Month_Long_StandAlone_9": "September",
+        "Month_Short_1": "Jan",
+        "Month_Short_10": "Okt",
+        "Month_Short_11": "Nov",
+        "Month_Short_12": "Des",
+        "Month_Short_2": "Feb",
+        "Month_Short_3": "Mar",
+        "Month_Short_4": "Apr",
+        "Month_Short_5": "Mei",
+        "Month_Short_6": "Jun",
+        "Month_Short_7": "Jul",
+        "Month_Short_8": "Agt",
+        "Month_Short_9": "Sep",
+        "Month_Short_StandAlone_1": "Jan",
+        "Month_Short_StandAlone_10": "Okt",
+        "Month_Short_StandAlone_11": "Nov",
+        "Month_Short_StandAlone_12": "Des",
+        "Month_Short_StandAlone_2": "Feb",
+        "Month_Short_StandAlone_3": "Mar",
+        "Month_Short_StandAlone_4": "Apr",
+        "Month_Short_StandAlone_5": "Mei",
+        "Month_Short_StandAlone_6": "Jun",
+        "Month_Short_StandAlone_7": "Jul",
+        "Month_Short_StandAlone_8": "Agt",
+        "Month_Short_StandAlone_9": "Sep",
         "NDays": "%s hari",
         "NHoursShort": "%s j",
         "NMinutes": "%s menit",
         "NMinutesShort": "%s mnt",
         "NSeconds": "%s detik",
         "NSecondsShort": "%s dtk",
+        "NumberFormatNumber": "#,##0.###",
+        "NumberFormatPercent": "#,##0%",
+        "NumberSymbolDecimal": ",",
+        "NumberSymbolGroup": ".",
+        "NumberSymbolMinus": "-",
+        "NumberSymbolPercent": "%",
+        "NumberSymbolPlus": "+",
         "OneDay": "1 hari",
         "OneMinute": "1 menit",
         "OneMinuteShort": "1 mnt",
-        "OriginalLanguageName": "Bahasa Indonesia",
+        "OriginalLanguageName": "Indonesia",
```

Comment:
That is what CLDR returns. They have done an update within the last month. So some strings might have changed